scripts/jenkins/mkcloud/openstack-mkcloud.prep.sh

```diff
@@ -6,6 +6,9 @@ shopt -s extglob
 mkdir -p empty
 rsync -r --delete empty/ ./
 
+if [ ! -v github_pr ] ; then
```

Comment:
github_pr is missing the $ , isn't it? 
`$github_pr`

Reply:
No, test -v takes a variable name, not the variable content.